app/src/main/java/ly/loud/loudly/ui/new_post/NewPostFragment.java

```diff
@@ -100,6 +110,23 @@ public void showGalleryError() {
 
     }
 
+    /**
+     * Triggered by presenter when loudlyPost uploaded to one more network.
+     */
+    @Override
+    public void onPostUploadingProgress(@NonNull LoudlyPost loudlyPost) {
+        // Just notify activity about progress
+        ((NewPostFragmentInteractions) getActivity()).onPostUploadProgress(loudlyPost);
```

Comment:
Here you may got ClassCastException, I we should use setter of callback (as in other fragments)